localstack/services/events/models.py

```diff
@@ -83,11 +86,16 @@ def __post_init__(self):
 
 EventBusDict = dict[EventBusName, EventBus]
 
+TagDict = dict[Arn, Tag]
+
 
 class EventsStore(BaseStore):
     # Map of eventbus names to eventbus objects. The name MUST be unique per account and region (works with AccountRegionBundle)
     event_buses: EventBusDict = LocalAttribute(default=dict)
 
+    # Maps resource ARN to tags
+    TAGS: TagDict = CrossRegionAttribute(default=dict)
```

Comment:
We have a `TaggingService` that implements tagging transparently (noticed by looking into DMS models as one of our new implementations). Is that applicable here or does different handling based on the resource type require a special case?

Reply:
used TaggingService in: https://github.com/localstack/localstack/pull/10840/commits/df9b76c5dadd4ba6b2496aacfc21bc1b8bc0fe85

The only issue I see is, that the pattern other services use is to store the TagginService objects in the store, which violates a bit of what we discussed a couple of weeks ago that data and functions should be strictly separated. This comes back to the issue that the AcountRegionBundle needs additional logic that handles this separation which would make a lot of providers cleaner.

Reply:
Very neat ✨
IMHO, that's a compromise worth taking vs. re-implementing the same thing for every service. The likelihood of a major change in tagging is very low.